libs/wire-api-federation/wire-api-federation.cabal

```diff
@@ -92,6 +92,7 @@ library
     , http-media
     , http-types
     , http2
```

Comment:
do we still want to depend on this?  or is there stuff we can re-export from http2-manager and then drop this (direct) dependency?  is it worth asking these questions?

Reply:
IMO, its ok to let people explicitly depend on this. This way we don't have to keep updating this library because the other one changed.